DigitalLearningSolutions.Web/Views/Home/Products.cshtml

```diff
@@ -0,0 +1,48 @@
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.Home
+@model LandingPageViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/products.css")" />
+

```

Comment:
Ah - we probably want to set the page title here & in the Learning Content pages :)